hvplot/util.py

```diff
@@ -121,6 +121,7 @@ def proj_to_cartopy(proj):
     import cartopy.crs as ccrs
     try:
         from osgeo import osr
+        getattr(osr, "UseExceptions", lambda: None)()
```

Comment:
Do you think there's a risk this breaks users code if they use GDAL on their side together with hvPlot? I'm not sure what it means to "use exceptions"?